src/workloads/scale/CreateDropView.yml

```diff
@@ -0,0 +1,81 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload loads a collection with documents and then repeatedly
+  create and drops a view on that collection.
+Keywords:
+- view
+- create
+- drop
+
+dbname: &dbname viewDB
+DocumentCount: &NumDocs 100000
+CollectionCount: &NumColls 1
+
+
+Actors:
+- Name: Setup
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    BatchSize: 100
+    Threads: 1
+    DocumentCount: *NumDocs
+    Database: *dbname
+    CollectionCount: *NumColls
+    Document: &doc
+      id: {^RandomInt: {min: 0, max: *NumDocs}}
+      a: {^RandomInt: {min: 0, max: *NumDocs}}
+      c: &string {^RandomString: {length: 50}}  # Adjust this so the doc comes out as 100 B.
```

Comment:
Is that comment correct? Are docs coming out do 100 B?

Reply:
I did some poking with [dataSize()](https://www.mongodb.com/docs/manual/reference/method/db.collection.dataSize/), and the comment is off. Each document comes out to **103** bytes. I think the size breaks down as follows: 
- 22 bytes to store _id as an objectId type 
- 23 bytes to store the ints, 8 bytes each for the 2 randInts up to 100K, 4 bytes for the 2 sets of “:” and “,”, and 3 bytes for the key character values “i”, “d”, and “a” 
- 58 bytes to store the string: 8 + length of 50 bytes

I got rid of the comment because I’m not particular about the document size. Originally I had kept this comment form where I got the loader form, MixedWorkloadsGenny.yml. Let me know if you think I should make any change to the comment in that file. 
I also got rid of the indexes. Creating the view doesn’t actually run the pipeline, so the indexes had no effect here.

Reply:
Awesome. Thanks.